### ISSUE TITLE
Could not find cursor theme Manjaro Linux

### ISSUE THREAD

**Issue by Supatier** [godotengine/godot#8908]
**Manjaro Linux (4.11-ck) Godot 2.1.3**

**Issue description:**
I'm using Godot from Steam. And this error comes out:
`WARNING: initialize: Could not find cursor theme
   At: platform/x11/os_x11.cpp:346.`

**Steps to reproduce:**
`abc@abc-pc ..am/steam/steamapps/common/Godot Engine % ./godot.x11.opt.tools.64 `

Is it an X error or some thing else, i'm not sure.
Any help would be appreciated...

**Comment by eon-s** [comment]
Here are other reports of the same issue, seems to be related with some desktop environments https://godotengine.org/qa/14171/how-to-fix-cursor-theme-error-on-linux

**Comment by Hinsbart** [comment]
@Supatier That's "just" a warning, not a real error. Most of the time you can safely ignore it.
Is there anything not working as expected related to the mouse cursor? e.g, does it have a different theme/not look how it's supposed to?

**Comment by Supatier** [comment]
@Hinsbart Everything works fine, but what does the error mean?

**Comment by Hinsbart** [comment]
@Supatier it means that we failed to get information about the cursor theme currently used by the X server and just use the default one. However, for the average user this information isn't useful at all (imo), especially since X11 is such a fragmented environment.. every window-manager seems to handle things a bit differently than the next. During my tests, I found that on some WMs this method will always fail.
That's why I recently removed the warning in the master branch.

**TL;DR** "Not much"
